sass/directives/_headers.scss

```diff
@@ -44,30 +24,58 @@
 
     @media only screen and (max-width: $small-screen-max) {
       position: relative;
+      padding-top: $header-top-bottom-padding;
+      padding-bottom: $header-top-bottom-padding;
     }
 
     h1 {
       margin-top: 0;
+      margin-bottom: $header-headline-bottom-margin;
       line-height: normal;
-      color: $hero-text-color-dark;
+      color: $hero-text-color;
+
+      @media only screen and (max-width: $small-screen-max) {
+        margin-bottom: 0;
+      }
     }
 
     p {
+      max-width: 28em;
       margin-bottom: 0;
       font-size: $hero-text-size;
-      color: $hero-text-color-dark;
+      color: $hero-text-color;
     }
   }
 }
 
-@mixin header-hero {
+@mixin header-hero--home($background-image-large, $background-image-medium: nil) {
   position: relative;
+  min-height: $header-height-large;
   overflow: hidden;
   background-color: $base-background-color;
+  background-image: url(asset_path(#{$background-image-large}));
+  background-size: auto $header-height-large;
+  background-position: right bottom;
+  background-repeat: repeat no-repeat;
+
```

Comment:
My IDE/linters/etc balk when I don't condense, but Sass compiles this down as efficiently as possible for us, and the condensed syntax is highly unreadable to a non-frontend engineer like myself. I've waffled on this a lot in the past, but now I'd like to push for us to adopt the more expanded syntax as @arelia has it here. Neither way is 'wrong', but we should definitely pick one.